### DISCUSSION TITLE
Large Language Models Are Zero-Shot Problem Solvers — Just Like Modern Computers | Tim Z. Xiao

### DISCUSSION THREAD

**Discussion by giscus** [timxzz/timxzz.github.io#4] [Comments]
# Large Language Models Are Zero-Shot Problem Solvers — Just Like Modern Computers | Tim Z. Xiao

Are large language models going to redefine the next generation of computers?

http://timx.me/blog/2023/computers-vs-llms/

**Comment by zc-alexfan** [comment]
有点意思

**Comment by Mikumikunisiteageru** [comment]
New bee

**Comment by JINKEHE** [comment]
牛蛙

**Comment by wangyongqiang2** [comment]
肖老板得闲来了解下国内人工智能发展情况啊！！！

**Comment by timxzz** [comment]
**NEW!!**
 We are please to announce that this article is now published at _Harvard Data Science Review (HDSR)_ [Issue 7.3](https://hdsr.mitpress.mit.edu/volume7issue3)! 

Please check out the new version there: [https://hdsr.mitpress.mit.edu/pub/42vjucmq/](https://hdsr.mitpress.mit.edu/pub/42vjucmq/)!

Best,
Tim Xiao
